modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -664,7 +666,16 @@ public ObjectEntry updateObjectEntry(
 					dtoConverterContext.getLocale()),
 				_createServiceContext(
 					objectEntry.getProperties(),
-					dtoConverterContext.getUserId())));
+					dtoConverterContext.getUserId()));
+
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153117"))) {
+			_upsertNestedObjectEntries(
```

Comment:
This map should be only created when the feature flag is true, so we could move it inside the if clause. Furthermore, the `objectRelationshipMap` parameter is used only once, so maybe we could do it inline